electron/app/components/JSONView.tsx

```diff
@@ -20,6 +20,15 @@ const Body = styled.div`
     flex-grow: 1;
     overflow-y: auto;
   }
+  pre::-webkit-scrollbar {
+    width: 0px;
+    background: transparent;
+    display: none;
+  }
+  pre::-webkit-scrollbar-thumb {
+    width: 0px;
+    display: none;
+  }
 
```

Comment:
Does `overflow: hidden` (or `overflow-x: hidden`) not work?

Reply:
You can't scroll with `hidden`. I would like to style our own scrollbar. This is my solution until then.

Reply:
If this allows scrolling, I think it should display a scrollbar, especially since other things do. We can certainly add custom scrollbars later, but at the moment, I don't think it's obvious that the sidebar is scrollable instead of just truncated.

Reply:
I was actually looking at the sidebar, so my comment applies there, but I think the same applies to JSONView.

(As a side note, on develop, I have noticed some "glitchy" scrollbar behavior when the size of the sidebar adjusts, but that would be a separate issue that custom scrollbars wouldn't resolve.)